Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM public.ecr.aws/lambda/python:3.8
+
+COPY requirements.txt  .
+RUN  pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
+
+COPY mappings.py service.py "${LAMBDA_TASK_ROOT}"/
```

Comment:
These are the two files (_mappings.py_, _service.py_) needed to run the `handler` function at this point. Maybe it would be better to have a package/folder where all the source file(s) would be and not worrying about updating the dockerfile accordingly if another file is added. All in all, for now, I chose the easy path but I thought it is worth mentioning it.